pkgs/tools/security/vault/default.nix

```diff
@@ -7,27 +7,32 @@ let
     rev = "e2f59b64be1fa5430fa05c91b6274284de4ea77c";
     sha256 = "10m75rp3hy71wlmnd88grmpjhqy0pwb9m8wm19l0f463xla54frd";
   };
-in buildGoPackage rec {
+in stdenv.mkDerivation rec {
```

Comment:
The git commit is lost anyway (```fetchFromGitHub``` loses it), the problem is this [version_base.go](https://github.com/hashicorp/vault/blob/5d2d750d9a5771d321a918d80dd0f160941fde68/version/version_base.go) is not compiled by ```buildGoPackage``` and ```"0.7.3"``` is lost